.circleci/docker/cloudbuild.yaml

```diff
@@ -6,15 +6,15 @@ steps:
   args: [
```

Comment:
Where is this file used?

Reply:
I used to create a manual trigger on GCP to build the image. I don't think it is used anywhere else

Reply:
I was wondering why we need to change this file.

Reply:
If it's not used, you can drop the change in this file.